### ISSUE TITLE
PR_END_OF_FILE_ERROR when using https for Server

### ISSUE THREAD

**Issue by mpricu** [stamparm/maltrail#16217]
Hi, I tried to implement https to connect to the Server component. I modified the maltrail.conf file and I also generated a new certificate based on the indications. The server looks like started on https, but the connection with the Firefox gave me the PR_END_OF_FILE_ERROR error. Using curl I noticed the following error:
OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to 0.0.0.0:8338 .

Please help me.

**Comment by MikhailKasimov** [comment]
Hello!

Is this info applicable for your case? --> https://www.elegantthemes.com/blog/wordpress/how-to-fix-the-pr_end_of_file_error-secure-connection-error

**Comment by mpricu** [comment]
Hi Mikhail,

Unfortunately that info is not applicable to me since I tried with 3 browsers (Firefox, Google Chrome and Lynx ) on 3 different OSes (Ubuntu 20.04 LTS, Centos 7 and Kali Linux 2021.1) and none were working. I attached an image where you can see the TLS connection with the Server is the problem. When I connected to the Google everything was fine, but in the Maltrail case the server did not respond correctly (it didn't say hello).
Thank you for your help.
![tls server handshake error](https://user-images.githubusercontent.com/40834170/116421970-fb77aa00-a847-11eb-95be-138373bbd6b5.jpg)

**Comment by stamparm** [comment]
![1](https://user-images.githubusercontent.com/921555/116428532-4a701000-a845-11eb-9e39-a15b979e1242.png)
![2](https://user-images.githubusercontent.com/921555/116428638-64115780-a845-11eb-96e0-01be14a81758.png)

**Comment by stamparm** [comment]
Now I see. Python3 seems to have a problem

**Comment by mpricu** [comment]
Hi,

I have the following Python versions on the OSes I tested:

Kali Linux - Python version 3.9.2
Ubuntu - Python version 3.8.5
Centos 7 - Python version 2.7.5

All Oses are vanilla installations. For Ubuntu and Kali I had to uninstall Python version 2 to be able to start Sensor and Server Maltrail, so Matrail is working with Python 3. On Centos 7, Maltrail is working with Python 2.7 .

Thank you for your help.
